packages/cell-toolbar/src/celltoolbartracker.ts

```diff
@@ -152,6 +153,15 @@ export class CellToolbarTracker implements IDisposable {
     return this._isDisposed;
   }
 
+  get visible(): boolean {
```

Comment:
- 1) Should it be `visible` or `enabled`? As a future user of this API I would expect `visible` to tell me something about the visbility when the toolbar is hidden due to the overlap, but it would not
- 2) Worth adding a docstring?